Discord/Commands/listAnon.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async function (
+  /** module:"discord.js".Message */ msg,
+  currentUserAnonymousChannels
+) {
+  if(Object.keys(currentUserAnonymousChannels).length === 0) {
+    msg.reply("Il n'y a pas de channels anonymes disponibles").catch(console.error);
+  }
+  else {
+    msg.reply("Les channels disponibles sont : "+Object.keys(currentUserAnonymousChannels).join(", "));
```

Comment:
```suggestion
    msg.reply(`Les channels disponibles sont : ${Object.keys(currentUserAnonymousChannels).join(", ")}`);
```

Je t'en fais qu'un mais je te conseilles de le faire pour tous. Tu peux automatiser cela avec Prettier

Reply:
fait